pkgs/jovian-greeter/greeter.py

```diff
@@ -166,16 +178,19 @@ def _consume_session(self) -> Optional[str]:
         return next_session
 
     def _find_sessions(self, sessions: Iterable[str]) -> Optional[Session]:
-        for data_dir in self.xdg_data_dirs + [ '/usr/share' ]:
+        for data_dir in self.xdg_data_dirs + [ '/usr/share', '/run/current-system/sw/share' ]:
             data_dir = Path(data_dir)
             for session in sessions:
+                logging.debug("Examining session: {}".format(session))
                 desktop_file = f'{session}.desktop'
                 wayland_session = data_dir.joinpath('wayland-sessions').joinpath(desktop_file)
                 x_session = data_dir.joinpath('xsessions').joinpath(desktop_file)
 
+                logging.debug("Checking Wayland session path: {}".format(wayland_session))
```

Comment:
I'd rather just log on error here, i.e. if session does _not_ exist, log that at WARNING (or even ERROR).

Reply:
The Wayland session path that it found is useful information for debugging, tho. It's generally a good idea to log I/O paths as TRACE or DEBUG IMO -- as the moment your program is touching other programs/file structures, making the interface points clear can make a lot of confusing errors more obvious.

If you're worried about cluttering up the log files, maybe it shouldn't default at log level DEBuG?

Reply:
I don't think it's particularly useful in 99.9% of cases.